hawkbit-repository/hawkbit-repository-jpa/src/test/java/org/eclipse/hawkbit/repository/jpa/TargetManagementTest.java

```diff
@@ -336,7 +364,7 @@ public void singleTargetIsInsertedIntoRepo() throws Exception {
         assertThat(target).as("Target compared with saved target").isEqualTo(savedTarget);
 
         savedTarget.setDescription("changed description");
-        Thread.sleep(1);
```

Comment:
Oh, the sleep probably made sense because of this assertion
```
assertThat(createdAt).as("CreatedAt compared with saved modifiedAt")
                .isNotEqualTo(savedTarget.getLastModifiedAt());
        assertThat(modifiedAt).as("ModifiedAt compared with saved modifiedAt")
                .isNotEqualTo(savedTarget.getLastModifiedAt());
```
It's a check about `notEqual` and maybe on a fast machine it could be equal?